core/controllers/contributor_dashboard_admin.py

```diff
@@ -516,7 +516,7 @@ class ContributorDashboardAdminStatsHandlerNormalizedPayloadDict(TypedDict):
     language_code: Optional[str]
     sort_by: Optional[str]
     topic_ids: Optional[List[str]]
-    max_days_since_last_activity: Optional[int]
+    last_date: str
```

Comment:
last_date is not a very intuitive name. Suppose you haven't been working on this handler for the last few weeks, would you understand what this parameter represents without looking into the implementation?

start_date would be a more canonical name.

Reply:
done